.github/workflows/Windows-pack.yml

```diff
@@ -2,15 +2,15 @@ name: Packaging(Windows)
 
 on:
   push:
-    branches: 
-      - master
+    branches:
+      - master*
     paths-ignore:
       - 'README.md'
       - 'LICENSE'
 
   pull_request:
     branches:
-      - master
+      - master*
```

Comment:
wildcard?

Reply:
I did it for testing, sometimes I need to check installation without merging it master. For example "master_test".